### ISSUE TITLE
More of a question for Fonyo than an issue. 

### ISSUE THREAD

**Issue by cmbconcretefire** [aaronsace/MultiPoolMiner#472]
Instead of re-writing everything, I will just copy and paste what I already wrote here. Thanks for the help Fonyo. (and anyone else who wants to help) 

In my bat file, when you get to the pools part, I have > miningpoolhub, miningpoolcoinhub. First, what is the difference between miningpoolhub and miningpoolcoinhub? I removed zpool and nicehash. I removed those 2 for 2 reasons. As I mentioned before, when I removed zpool, I removed the need for a BTC address in the bat file and that made it a little easier on me at first just to get it up and running. I removed nicehash because I already have nicehash as a completely separate miner. 

Question: I have since added a btc address back to the bat file. 1) Should I add zpool back to the list? And I went to zpool's website and it looks like their config (bat) file is structured differently. So, if I do add zpool back, do I need to do anything to the bat file OTHER THAN just adding algo's they have that MPH doesn't?  And should I add nicehash back too? 

I am including a screenshot because I have 2 questions about that too: https://imgur.com/a/KhAsR

Notice the "FireiceNvidia 1,2,4, & 5? Where is 0 and 3? I also have the same problem on my 1070 rig. But it's only missing 1 card. 
Why did that not benchmark all 6 of my cards and is there a way I can correct that without having to start from scratch in benchmarking? 

And the last question regarding that picture: I think I understand what's going on under the "Launched Command." And below that the "Miner" part I think I understand. But to the right of "Miner", I know what BTC means and USD but as far as the %'s and those numbers.....no idea what I'm looking at. > But I am going to read the FAQ again.

**Comment by fonyo** [comment]
Don't worry about miningpoolhub and miningpoolhubcoins (the truth is I can't really remember what the difference was, something about individual coins, maybe @aaronsace could explain or you can have a look at the code in the Pools folder).

_So as the README states_, you add the algorithms you want to mine and this command will override your pool settings if not done correctly (or intentionally done incorrectly). I will give you two examples:
You set the poolname to MPH and zpool, and have Pascal listed after the algorithm command. Pascal will be mined on nicehash (when profitable) as it is only available on that pool.
You set the poolname to zpool, and have ethash listed. You will mine on zpool essentially but mine the most profitable ethash coin on MPH or nicehash when ethash algo is the most profitable.
I hope this makes sense. Please bear in mind the README is there to guide people but its scope does not and cannot cover teaching people basic stuff about command line applications. 

We have recently had a discussion about spreading the earnings, because that is actually what you do when mining on multiple pools. Pools have different payout methods (auto/manual/both), and different fees and rules etc about the same. If you are happy with this, then do it. I, for one, would not recommend doing so as if you are not reaching the minimum payout threshold for a longer while, you may lose your earnings.

For the reasons above, I would be very careful adding extra pools, especially being a new user. Try to restrict your mining to a single pool and less algos, then you can always expand and try things out once you have an idea of how much you can earn with your rig and compare pools, algos etc. Keep it simple but at the end of the day it is your decision what you do with your rig.

Fireice: there are, indeed, 6 of them. Some of them might have failed during benchmarking and then failed miners (or different configs of the same miner in this case) will get de-listed from MPM. You will only see the ones that are viable and working with your rig.

As per README (not AGAIN?!), you can delete individual files from the stats folder to reset each miners (configs) benchmark data and force MPM to rebenchmark it.

And the last paragraph of your queries, the answer is in the README again. FAQ#5

**Comment by cmbconcretefire** [comment]
I just wanted to say thank you. As someone who is brand new to MPH and the MultipolMiner, I think I was over-thinking it. I told you I had read the FAQ, and that I would read it again. As I DID read it again, I realized every one of my questions was answered right there in the FAQ. My point being, when you are new at something and you read the "manual" , some things may not make sense until you actually play around with it a little bit. Then you read the "manual" (FAQ) again, and suddenly everything makes sense. You must think I'm an idiot. Lol. I assure you, everything makes sense now.

**Comment by fonyo** [comment]
Yes, I know, and that's why I am helping. I have been mining for many years and I still learn new things every day. It can be pretty overwhelming when you start, especially when it comes to more advanced multipool/multialgo software such as this one. We didn't have these years ago, you mined a coin and that was it. It has been bumpy ride but that makes it fun sometimes, and you are ripping your hair out other times. :)

**Comment by fonyo** [comment]
Closing this, post a new thread if you have other issues/queries.
